Example-CustomUserGroups-REXX/README.md

```diff
@@ -0,0 +1,500 @@
+
+Note: The Custom User Group being displayed and modified are located on the Support Element (SE)
+
+## System Prep work
+- Store USRGRP01 into a data set of at least fb80
+- Ensure your z/OS user ID has at least READ and CONTROL access to the following FACILITY Class Profile
+    - HWI.TARGET.netid.nau
+
+    <p>where netid.nau represents the 3– to 17– character SNA name of the particular CPC. </p>
+
+## Invocation
+**Syntax**:
+```
+USRGRP01 [-C CPCName] [-L LPARName] [-G CustomUserGroup] [-A Action {ADD, REMOVE, LIST}] [-I] [-V] [-H]
+```
+
+ Optional Input Parameters:<br>
+  - CPCName  - The name of the CPC to query, the default is the local CPC if not specified or provided<br>
+  - LPARName - The name of the LPAR to query and target, the default is
+                 the local LPAR if not specified or provided<br>
+  - UserGroupName - The name of the Custom User Group to target<br>
+  - Action        - The action to take against the Custom User Group
+                    limited to ADD, REMOVE, or LIST<br>
+  - -I            - indicate running in ISV REXX environment, default
+                    is TSO/E<br>
+  - -V            - turn on additional verbose JSON tracing<br>
+  - -H            - Display sample parameters and how to invoke the sample<br>
+
+**Note**: If no arguments are supplied then the sample will default to listing all of the Custom User Groups on the local CPC.
+
+**Sample Invocation in TSO:**
+
+USRGRP01 has been copied into data set HWI.USER.REXX <br>
+
+  List all Custom User Groups on the specified CPC:<br>
+  - Local CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)'
+    ```
+
+  - Remote CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)' '-C CPC1 -L LPAR1'
+    ```
+
+  List the members of the TEST Custom User Group:
+  - Local CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)' '-G TEST'
+    ```
+
+  - Remote CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)' '-C CPC1 -L LPAR1 -G TEST'
+    ```
+
+  Add an LPAR as a new member to the TEST Custom User Group
+
+  - Local CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)' '-G TEST -A ADD'
+    ```
+
+  - Remote CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)' '-C CPC1 -L LPAR1 -G TEST -A ADD'
+    ```
+
+  Remove an LPAR member from the TEST Custom User Group
+
+  - Local CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)' '-G TEST -A REMOVE'
+    ```
+
+  - Remote CPC and LPAR:
+    ```
+    ex 'HWI.USER.REXX(USRGRP01)' '-C CPC1 -L LPAR1 -G TEST -A REMOVE'
+    ```
+
+**Sample Batch Invocation via JCL:**
+USRGRP01 has been copied into data set HWI.USER.REXX
+
+```
+ //USRGRP01 JOB ,
+ // CLASS=J,NOTIFY=&SYSUID,MSGLEVEL=1,
+ //  MSGCLASS=H,REGION=0M,TIME=1440
+ //STEP1    EXEC PGM=IKJEFT01,DYNAMNBR=20
+ //SYSUDUMP DD SYSOUT=(H,,STD)
+ //SYSTSPRT DD SYSOUT=(H,,STD)
+ //SYSTSIN  DD * UB
+ PROFILE NOPREFIX
+ EX 'HWI.USER.REXX(USRGRP01)'  -
+ 'CPC1 LPAR1 TESTGRP ADD'

```

Comment:
update this invocation

Reply:
?

Reply:
if you try that right now.... getArgs will fail you